src/SonarQube.Client.Tests/SonarQubeService_GetSuppressedIssuesAsync.cs

```diff
@@ -381,7 +381,7 @@ public async Task GetSuppressedIssuesAsync_From_7_20_BranchIsNotSpecified_Branch
             // Branch is null/empty => should not be passed
             var actualRequests = messageHandler.GetSendAsyncRequests();
             actualRequests.Should().HaveCount(3);
-            actualRequests.Any(x => x.RequestUri.Query.Contains("branch")).Should().BeFalse();
+            actualRequests.Should().NotContain(x => x.RequestUri.Query.Contains("branch"));
```

Comment:
Changes suggested by the Fluent analyzers. I'm not convinced they are easier to read, but there you go.